packages/lesswrong/components/sequences/ChapterTitle.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { forumTypeSetting } from '../../lib/instanceSettings';
+import { registerComponent } from '../../lib/vulcan-lib';
+
+const isEAForum = forumTypeSetting.get() === "EAForum"
+
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    ...theme.typography.chapterTitle,
+    margin: "1em 0",
+  }
+});
+
+const ChapterTitle = ({classes, title}: {
+  classes: ClassesType,
+  title?: string
+}) => {
+  if (!title) return null
```

Comment:
I think I'd rather externalize this and require the caller to check if they have a title before trying to render the component